src/main/java/org/jabref/gui/JabRefFrame.java

```diff
@@ -1422,4 +1463,22 @@ private void updateTexts(UndoChangeEvent event) {
             */
         }
     }
+
+    private void copyGroupTreeNode(LibraryTab libraryTabCasted, Optional<GroupTreeNode> parent, Optional<GroupTreeNode> groupTreeNodeToCopy, List<BibEntry> allEntries) {
```

Comment:
Is passing *allEntries* here really necessary?

Reply:
But still, you don't have to pass it as an argument. You can create a local variable inside the copy method and have it hold all entries.